config/test.exs

```diff
@@ -8,7 +8,7 @@ config :plenario, PlenarioWeb.Endpoint,
 
 
 # Print only warnings and errors during test
-config :logger, level: :warn, metadata: [:request_id]
+config :logger, level: :error, metadata: [:request_id]
```

Comment:
... because I'm sick of having my test output polluted with Sentry warnings

Reply:
I don't think it's a good idea to address this by restricting the logger. There have been warnings that have helper me out a lot, and even though this gets rid of the warnings in test, they're still present (and equally obnoxious) when playing around in dev. The right way, I believe, is to override the `handle_errors` method described [here](https://hexdocs.pm/sentry/Sentry.Plug.html#content). Let's just make an issue for this for now.